src/util/instances.tsx

```diff
@@ -0,0 +1,23 @@
+import { LxdOperationResponse } from "types/operation";
+import { getInstanceName } from "./operations";
+import InstanceLink from "pages/instances/InstanceLink";
+import React from "react";
+
+export const generateInstanceLink = (args: {
+  instanceName?: string;
+  operation?: LxdOperationResponse;
+  project?: string;
```

Comment:
I think this interface should be more concrete. Currently, you can give it either an instanceName or an operation. I think that should be two functions, one giving you a link from a string and one for an operation. Wdyt?

Reply:
yeah that sounds good, I will put the operation link generator in the `util/operations` file